kalite/distributed/static/css/distributed/bootstrap-overrides.css

```diff
@@ -436,7 +436,7 @@ footer {
 }
 
 @media (max-width: 454px) {
-    .form-control {
+    #search_box {
```

Comment:
Double checked, and it doesn't seem like others are negatively affected by this. Merging.